examples/nine-pt-stencil-solver/doc/intro.dox

```diff
@@ -27,7 +27,7 @@ On any node, where neighbor is on the border, the neighbor is replaced with a
 example a node with a neighborhood of only edge nodes may look like this
 
 \sum_{a,b=-1}^(1,0) stencil(a,b) * u_{(i+a,j+b} = -f_k h^2 - \sum_{a=-1}^1
-stencil(a,1) * * u_{(i+a,j+1}
+stencil(a,1) * u_{(i+a,j+1}
```

Comment:
@pratikvn is right, you are missing `$` signs around your math equations (around most of them, not just here).